src/main/java/de/tum/in/www1/artemis/web/rest/errors/StudentsAlreadyAssignedException.java

```diff
@@ -24,7 +23,7 @@
     private static final String ERROR_KEY = "studentsAlreadyAssignedToTeams";
 
     public StudentsAlreadyAssignedException(List<Pair<User, Team>> conflicts) {
-        super(ErrorConstants.STUDENT_ALREADY_ASSIGNED_TYPE, "Student(s) have already been assigned to other teams.", TeamResource.ENTITY_NAME, ERROR_KEY, getParameters(conflicts));
+        super(ErrorConstants.STUDENT_ALREADY_ASSIGNED_TYPE, "Student(s) have already been assigned to other teams.", "team", ERROR_KEY, getParameters(conflicts));
```

Comment:
Why this change? I think the usage of constants makes here sense in general.

Reply:
Al of the entities and repositories are included in the source sets for each microservice. TeamRepository has a dependency to this exception, therefore this exception also needs to be included. And if I include in the current state I also need to include the TeamResource + other dependencies coming from this class. This will end up in the TeamResource endpoints being accessible from the user management microservice and I think this doesn't make sense

Reply:
I agree, however, this solution is error-prone.
Would it be possible to extract the constants you require to dedicated files that get imported by the mainframe and your microservice? Then you only need to import one class.